.github/workflows/create-release-branch.yaml

```diff
@@ -96,11 +93,9 @@ jobs:
           AZURE_CORE_ONLY_SHOW_ERRORS: True
         run: make test-kubernetes
       - name: create branch
-        uses: peterjgrainger/action-create-branch@v2.0.1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          branch: '${{ env.RELEASE_BRANCH }}'
+        run: |
+          git branch release-${{github.event.inputs.release_version}}

```

Comment:
Did you want to use git branch -c such that the original branch is maintained (not renamed)?

What this does is lose the current branch and replace it with the release branch.  With the `-c` option, it would keep the source branch (leave it alone) and create the release branch as the same ref hash/log.

It all depends on the behavior you want as to the source branch - does it get deleted or not?

Reply:
I'm actually simply trying to create a *new* branch from the commit of the source branch, and push it to the Azure/aks-engine remote. This didn't work, so I'm trying this:

https://github.com/Azure/aks-engine/pull/4576